.github/workflows/tests.yml

```diff
@@ -56,7 +56,7 @@ jobs:
     runs-on: windows-2022
     steps:
       - name: Install Crystal
-        uses: crystal-lang/install-crystal@v1
+        uses: crystal-lang/install-crystal@e4c9d197a5d1fa8329b55dadf309d06050ef5d07
```

Comment:
Maybe also use this in .github/workflows/doc-generator.yml?

Reply:
Sure